x/auth/client/cli/tx_multisign.go

```diff
@@ -127,7 +136,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 				}
 			}
 		}
-
+		// TODO how to deal with multisigPub and multisigSig ?
 		sigV2 := signingtypes.SignatureV2{
```

Comment:
@amaurym, Following our last conversation, you pointed out we no longer use Amino outside `checkMigrate` in keyring.go. Just wondering if `makeMultiSignCmd()`  makes sense or it has to be deleted?